ads/google/a4a/utils.js

```diff
@@ -110,16 +113,23 @@ export function googleAdUrl(
 export function extractGoogleAdCreativeAndSignature(
```

Comment:
This is a bit of a misnomer given we now include size.  You don't have to do so in this PR but would be good to rename (perhaps extractAdResponse to match the return type?)